WordPress/WordPressHomeWidgetToday/Supporting Files/WordPressHomeWidgetTodayRelease-Alpha.entitlements

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.org.wordpress.alpha</string>
+	</array>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)3TMU3BH3NK.org.wordpress</string>
```

Comment:
I think the bundle prefix is different for alpha and internal builds (check the existing widget entitlements files) – `99KV9Z6BKV`. So not sure if it needs to match here and in the Internal entitlements?

Reply:
I think so too and good catch! Before messing up with entitlements, I'd kindly ask @jkmassel and @AliSoftware if there's anything specific that needs to be done here?

Reply:
Yep – good catch @frosty. This is fixed in 1cdff2b0!

Reply:
Thank you @jkmassel !!

Reply:
Great catch!!